### DISCUSSION TITLE
Sharing Discord channel links, problem

### DISCUSSION THREAD

**Discussion by holmbuar** [carbon-language/carbon-lang#2084] [General]
I kindly suggest not to share Discord channel links, because they don't work for users who has not done the process of clicking the Discord invitation link, and then being registered on the carbon-lang Discord server. 

The error message showing up for the uninvited is less than helpful, it seems.

Better to share link to Discord invites, and state the relevant channel name.

**Comment by sociop** [comment]
dont add me to this discussions

**Reply by jonmeow** [reply]
@qjnqjn You're seeing this due to your subscription settings. You need to [unsubscribe](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/managing-subscriptions-for-activity-on-github/managing-your-subscriptions).

**Comment by jonmeow** [comment]
The links are pretty helpful for those of us who are subscribed. Discord links are especially helpful for linking to particular conversations. It'd probably be helpful as a habit to mention how to subscribe to discord, but that's already in our readme and probably people won't remember to bring it up every time they post a link. I think it'd be nice if Discord's error were more helpful, but that's out of our control. (I have few ideas of how to make the experience better for everyone)

**Reply by ki37ki** [reply]
good fruitful work done. very helpful stuff!

**Comment by sk03d** [comment]
The discord link mentioned in the about is not working, am not able to accept the invite. Please help me out
